install_config/configuring_pipeline_execution.adoc

```diff
@@ -7,26 +7,32 @@
 :experimental:
 :toc: macro
 :toc-title:
+:prewrap!:
 
 toc::[]
 
 
 == Overview
 
-{product-title} utilizes Jenkins to execute continuous development pipelines.  To
-provide a seamless experience for users, the first time a user creates a build
-configuration using the xref:../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[pipeline]
-strategy type, a set of Jenkins related resources will be created within the user's project.
-By default, the system will look for a template named `*jenkins*` in the `*openshift*` namespace and
-instantiate it within the user's project.  {product-title} ships with a default template definition
-which will, upon instantiation, create the following in the user's project:
+{product-title} utilizes Jenkins to execute continuous development pipelines.
+To provide a seamless experience for users,
+the first time a user creates a build configuration using the
+xref:../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[Pipeline]
+strategy type,
+a set of Jenkins related resources will be created within the user's project.
+By default, the system will look for a template named `*jenkins*`
+in the `*openshift*` namespace and instantiate it within the user's project.
+{product-title} ships with a default template definition which will,
```

Comment:
@bparees Thanks for the clarification.

> assuming it was registered with the cluster at install time

I take it that the ones i _did_ see in the OSE 3.2 instance docs team plays with (i.e., `jenkins-ephemeral` and `jenkins-persistent`) were indeed registered with the cluster at install time.  As i don't have a 3.3 instance to play w/ (yet? -- @vikram-redhat), here are some followup questions:

- Does 3.3 also provide `jenkins-ephemeral` and `jenkins-persistent`?
  * If so, should they be mentioned in these docs?
  * If not, should their omission between 3.2 and 3.3 be mentioned in the release notes?
- Where in the source code can i look to see what templates are "by default registered at cluster install time"?